.github/contributing.md

```diff
@@ -0,0 +1,245 @@
+# Livtours Contribution Code Standards
+
+Livtours follows a series of operating procedures and conventions when contributing code to the codebase. These are outlined below.
+
+## Branch creation and naming conventions utilised
+
+Upon tackling a new issue in our codebase,
+
+1. Checkout and pull `dev` branch (ie. default branch)
+
+2. Create your new branch following the convention as follows:
+
+`{ISSUE-TYPE}/{ISSUE-ID}/{ISSUE-DESCRIPTION}`
+
+where:
+
+- ISSUE-TYPE: One of the issue type label group prefixes specified in the `GH labels`
+      documentation or currently in place on Github.
+
+      FEAT = type:feature
+      BUG = type:bug
+      CHORE = type:chore
+      REF = type:refactor
+      PROJ = type:project
+
+      (eg. FEAT, BUG, CHORE, DOCS, TEST, REF, PROJ, etc..)
+
+<br>
+
+- ISSUE-ID: This the id automatically generated when the issue is created on Github.
+
+  The id must:
+   - be unique
+   - contain a precursory hashtag
+
+  This can be found here:
+
+  ![](./resources/doc-images/issue-id.png)
+
+<br>
+
+- ISSUE-DESCRIPTION: This is a description of the issue.
+
+  It must be:
+   - clear
+   - concise
+   - sensical
+   - self-explanatory
+
+  It must contain:
+   - ONLY lowercase characters
+   - "-" as substitutes for spaces in multi-word descriptions
+
+  <br>
+
+  A few examples of sample branches:
+    1. `FEAT/#567/format-paragraphs-acc-page`
+    2. `REF/#78/modify-cart-request-logic`
+    3. `DOCS/#69/add-organisation-level-code-contribution-file`
+
+<br>
+
+## Pre-committing to Github
+
+Before committing our code to Github, there are a few considerations we need to take into account
+
+- Final code optimization and examination
+- Linting and formatting
+- Testing
+- Documentation
+
+To enforce a uniform standard of code across our codebase, we have implemented CI and pre-commit hooks to perform these operations on our code according to predefined rules and commonly utilised ruleset settings.
+
+CI Github workflows and use of pre-commit hooks by `Husky` make this possible.
+
+- A final review of your code should be performed to reduce code review overhead by other contributors
+
+  Aspects of code you should check include:
+
+    - Newlines on last line of all files
+    - Trailing spaces and log statements are removed
+    - Necessary types and documentation are present
+    - Globalization of all textual content
```

Comment:
not necessarily. 

Out bluemoon platform, for example, is only in english and I'm not sure we want to globalise it

Reply:
Okay interesting, 

So, it would be more suitable to leave this line out or be a bit more specific and reword it to: 
` Globalization of all textual content (ie. When operating in repo responsible for customer facing platform)`?

Reply:
> Okay interesting,
> 
> So, it would be more suitable to leave this line out or be a bit more specific and reword it to: `Globalization of all textual content (ie. When operating in repo responsible for customer facing platform)`?

Maybe we can just put under a `LivTours`-specific subsection